be/src/exec/iceberg/iceberg_delete_builder.h

```diff
@@ -45,9 +47,10 @@ class IcebergDeleteBuilder {
             const std::shared_ptr<io::SharedBufferedInputStream>& shared_buffered_input_stream);
     Status fill_skip_rowids(const ChunkPtr& chunk) const;
 
-    std::set<int64_t>* _need_skip_rowids;
+    SkipRowsContextPtr _skip_rows_ctx;
     const HdfsScannerParams& _params;
     RuntimeState* _runtime_state;
+    roaring64_bitmap_t* _deletion_bitmap;
```

Comment:
Why not use roaring32_bitmap, the performance may be better.

Reply:
![image](https://github.com/user-attachments/assets/8af09410-5ce5-4637-a60f-2b4f6b2943cf)
because iceberg and delta lake use 64-bit integers